barretenberg/cpp/src/barretenberg/commitment_schemes/kzg/kzg.test.cpp

```diff
@@ -41,6 +41,42 @@ TYPED_TEST(KZGTest, single)
     EXPECT_EQ(this->vk()->pairing_check(pairing_points[0], pairing_points[1]), true);
 }
 
+/**
```

Comment:
We usually commit to and open univariate polynomials represented in the monomial basis. Here is a correct flow for the univariates given in the Lagrange basis.